### ISSUE TITLE
AttributeError: module 'string' has no attribute 'strip' when installing with Python3

### ISSUE THREAD

**Issue by lewismc** [pydap/pydap#23]
Hi folks, [as noted some time ago](https://groups.google.com/a/opendap.org/forum/#!msg/support/-_15rXI-4dI/vrd0_GvjCwAJ) Pydap cannot be installed using Python3.
This issue should fix the issue below.

```
Collecting pydap>=3.1.1 (from -r ocw-pip-dependencies.txt (line 3))
  Using cached Pydap-3.1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-bpljh_u8/pydap/setup.py", line 10, in <module>
        paver.tasks.main()
      File "paver-minilib.zip/paver/tasks.py", line 883, in main
      File "paver-minilib.zip/paver/tasks.py", line 851, in _launch_pavement
      File "pavement.py", line 31, in <module>
        classifiers=filter(None, map(string.strip, '''
    AttributeError: module 'string' has no attribute 'strip'
```

**Comment by lewismc** [comment]
This PR I am sending also ensures that all of the code is PEP8 compliant and upgrades to Python 3 syntax.

**Comment by lewismc** [comment]
Thiis issue has been addressed within issue #9
